frontend/pages/index.js

```diff
@@ -41,27 +41,27 @@ const Home = () => {
    * @param ModalValue -The current value of the state
    */
   /* eslint-disable no-unused-vars*/
```

Comment:
We actually don't want to disable unused vars. If a variable is not being used, it should not be included in the file, if it is intentionally unused (because you need the second parameter of a function, instead of the first for example) you can designate that using `_variableName`

Reply:
I originally needed that because it was erroring out at the second variable. doesnt seem to be anymore - perhaps because i changed it from a function to a const.